internal/apis/kfd/v1alpha2/eks/delete/distribution.go

```diff
@@ -251,11 +287,20 @@ func (d *Distribution) checkPendingResources() error {
 		p := time.NewTicker(dur)
 
 		if <-p.C; true {
-			errSvc = d.getLoadBalancers()
+			lbs, errSvc = d.kubeClient.GetLoadBalancers()
+			if errSvc == nil && len(lbs) > 0 {
+				errSvc = fmt.Errorf("%w: %v", errPendingResources, lbs)
+			}
 
-			errPv = d.getPersistentVolumes()
+			pvs, errPv = d.kubeClient.GetPersistentVolumes()
+			if errPv == nil && len(pvs) > 0 {
+				errPv = fmt.Errorf("%w: %v", errPendingResources, pvs)
+			}
 
-			errIgrs = d.getIngresses()
+			ingrs, errIgrs = d.kubeClient.GetIngresses()
+			if errIgrs == nil && len(ingrs) > 0 {
+				errIgrs = fmt.Errorf("%w: %v", errPendingResources, ingrs)
+			}
```

Comment:
we could think about running the three calls in parallel to speed up things a bit

Reply:
card created: https://github.com/sighupio/furyctl/issues/341